tests/suites/test_suite_psa_crypto.function

```diff
@@ -294,6 +294,18 @@ exit:
     ((void) 0)
 #endif /* MBEDTLS_PSA_CRYPTO_SE_C */
 
+uint64_t parse_hex_string(data_t *hex_string)
+{
+    uint64_t result = 0;
+    if (hex_string->len > 8) {
+        return 0;
```

Comment:
Rather than silently return some arbitrary value if the test data is bad, we should mark the test as failed (it's ok to do that in an auxiliary function). The function does have to return something (there's no exception/longjmp mechanism), so it's ok to go on and return some arbitrary value.

So:

```
TEST_LE_U(data->len, 8);
…
exit:
    return 0; // or whatever
```